letsencrypt/le_util.py

```diff
@@ -70,8 +70,10 @@ def unique_file(path, mode=0o777):
         try:
             file_d = os.open(fname, os.O_CREAT | os.O_EXCL | os.O_RDWR, mode)
             return os.fdopen(file_d, "w"), fname
-        except OSError:
-            pass
+        except OSError, e:
```

Comment:
`except OSError as error`, the above is deprecated syntax and `e` is too short variable name (pylint should complain)